projects/baby_driver/scripts/adc_read.py

```diff
@@ -0,0 +1,58 @@
+"""Python implementation of the adc_read function for Babydriver."""
+
+import can_util
+from gpio_port import GpioPort
+from message_defs import BabydriverMessageId
+
+NUM_PINS_PER_PORT = 16
+
+def adc_read(port, pin, raw):
```

Comment:
One thing you forgot: you need to `from adc_read import adc_read` and `from gpio_port import GpioPort` in the top level of `repl_setup.py` so that the REPL has access to the function without importing it.